packages/statistics/script/sex/importStatisticsNameSex.js

```diff
@@ -0,0 +1,62 @@
+require('@orbiting/backend-modules-env').config()
+const PgDb = require('@orbiting/backend-modules-base/lib/PgDb')
+const rw = require('rw')
+const { dsvFormat } = require('d3-dsv')
+const csvParse = dsvFormat(',').parse
+
+console.log('running import of first name statistics of BfS....')
+
+PgDb.connect()
+  .then(async (pgdb) => {
+    const dry = process.argv[2] === '--dry'
+    if (dry) {
+      console.log("dry run: this won't change anything")
+    }
+
+    const input = rw.readFileSync('/dev/stdin', 'utf8')
+    if (!input) {
+      throw new Error('You need to provide input on stdin')
+    }
+
+    const transaction = await pgdb.transactionBegin()
+
+    console.log('Truncate db table statisticsNameSex...')
+    await transaction.query(`TRUNCATE TABLE "statisticsNameSex"`)
```

Comment:
(This is more of a note, [would use Pogi here](https://pogi.readthedocs.io/API/PgTable/#truncate).)